lib/resque/scheduler.rb

```diff
@@ -206,11 +228,13 @@ def enqueue_next_item(timestamp)
 
       # Enqueues all delayed jobs for a timestamp
       def enqueue_delayed_items_for_timestamp(timestamp)
+        # CLIO: Add metric for how long it took to move an item from scheduler to Resque
+        datadog.histogram("resque.schedule.latency", Time.now.to_f - timestamp)
+
         item = nil
         loop do
           handle_shutdown do
-            # Continually check that it is still the master
-            item = enqueue_next_item(timestamp) if am_master
+            item = enqueue_next_item(timestamp)
```

Comment:
This appears to be another major change, we used to do this conditionaly only `if am_master`